app/js/components/questions/Show.react.spec.js

```diff
@@ -0,0 +1,28 @@
+import expect from 'expect';
+import React from 'react'; // eslint-disable-line no-unused-vars
+import {mount, shallow} from 'enzyme'; // eslint-disable-line no-unused-vars
+import Show from './Show.react';
+
+/**
+* @return {Func} A shallow dom for tests
+*/
+function setup(status) {
+  let props = {
+    tag: {id: 1, name: 'first tag'},
+    index: 1,
+    options: ['first pic', 'second pic'],
+    selectedStatus: status,
+    onShowClick: () => {}
+  };
+  return mount(<Show {...props} />);
+}
+
+describe('<Index />', () => {
+  it('renders the tag component', () => {
+    // const wrapper = setup();
```

Comment:
Commented out code shouldn't make it to production. also setup can also be placed inside a beforeEach :)